iphone/Maps/Categories/MWMCategory+PlacesCountTitle.swift

```diff
@@ -2,13 +2,15 @@ import Foundation
 
 extension BookmarkGroup {
   @objc func placesCountTitle() -> String {
-    if (bookmarksCount > 0 && trackCount > 0) ||
-      (bookmarksCount == 0 && trackCount == 0) {
-       return String(format: L("objects"), bookmarksCount + trackCount)
-    } else if (bookmarksCount > 0) {
-      return String(format: L("bookmarks_places"), bookmarksCount)
-    } else {
-       return String(format: L("tracks"), trackCount)
+    switch (bookmarksCount, trackCount) {
+      case (let bookmarks, let tracks) where bookmarks > 0 && tracks > 0:
+          return "\(String(format: L("bookmarks_places"), bookmarksCount)), \(String(format: L("tracks"), trackCount))"
+      case (let bookmarks, _) where bookmarks > 0:
+          return String(format: L("bookmarks_places"), bookmarks)
+      case (_, let tracks) where tracks > 0:
+          return String(format: L("tracks"), tracks)
+      default:
+          return String(format: L("objects"), bookmarksCount + trackCount)
```

Comment:
Switch is less readable here, and it also has lookingly unnecessary case. When is this default used?

```
if (bookmarksCount)
{
  if (tracksCount)
    return bookmarks + tracks;
  return bookmarks;
}
return tracks;
```